mount/mount_id_mapped.c

```diff
@@ -0,0 +1,83 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
```

Comment:
Should be Apache and should be written in Go?

Reply:
Sure. I wanted to prototype this quickly and based on https://github.com/brauner/mount-idmapped/blob/master/mount-idmapped.c, that's the reason why I used that license and language.